docs/admin/quick.rst

```diff
@@ -70,6 +169,8 @@ Installing on OpenShift 2
    
     For more information, including on how to retrieve the generated admin password, see :ref:`openshift`.
 
+ .. _adding-translation:
+
```

Comment:
docs/devel/translations.rst:7:duplicate label adding-translation, other instance in docs/admin/quick.rst, you should choose different label here.

Reply:
Okay, so that label is auto-added for the heading? Will it work if I just remove it? 
I have read that auto-label-adding for headings requires a plugin in Sphinx, and I wasn't sure which plugins you may have installed.

Reply:
AFAIK there are no automatic labels, so just choose something different in this document.

Reply:
Thanks!

Reply:
Done, hope it's going to work now. 

The reason for assuming auto-labelling is that there is no other instance of 'adding-translation' in the file, although the error message claims that there is.